### PR TITLE
Some fixes in the "faster command line tools" article

### PR DESCRIPTION
**Pull request by gangsterakato**

### CODE REVIEW

**Comment by dom96** [comment]
@euantorano please review.

File: jekyll/_posts/2017-05-25-faster-command-line-tools-in-nim.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Rather than explain the task again, I might as well quote the reference post tha
 >
 > With the first field as key, second field as value, the key with the max sum is B, with a total of 13.
 >
-> Fields are delimited by a TAB, and there may be any number of fields on a line. The file name and field numbers of the key and value are passed as command line arguments.
+> Fields are delimited by a *tabulation*, and there may be any number of fields on a line. The file name and field numbers of the key and value are passed as command line arguments.
```
Comment (euantorano) [inline jekyll/_posts/2017-05-25-faster-command-line-tools-in-nim.md]: This is a quote from the original D blog post, so it doesn't make sense to change the content.

File: jekyll/_posts/2017-05-25-faster-command-line-tools-in-nim.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ This is quite a simple program. All that we need to do is read a file, parse it
 Luckily, Nim's standard library has many useful modules to help with common tasks like these:
 
 - The [streams](https://nim-lang.org/docs/streams.html) module provides a `FileStream` type for reading from and writing to files.
-- The [parsecv](https://nim-lang.org/docs/parsecsv.html) module provides a simple high performance CSV parser.
+- The [parsecsv](https://nim-lang.org/docs/parsecsv.html) module provides a simple high performance CSV parser.
```
Comment (euantorano) [inline jekyll/_posts/2017-05-25-faster-command-line-tools-in-nim.md]: Good catch! I need to change this in the master copy.

File: jekyll/_posts/2017-05-25-faster-command-line-tools-in-nim.md [hunk 3]
```diff
@@ -258,9 +258,8 @@ I cleared out all build artefacts (such as the `*.o` files for D and the `nimcac
 
 It turns out that the DMD compiler is much quicker than the LDC compiler. Obviously all the optimisations that LLVM makes take a little while longer to complete.
 
-
```
Comment (euantorano) [inline jekyll/_posts/2017-05-25-faster-command-line-tools-in-nim.md]: Extra line was needed for formatting reasons on my blog, can be removed here, so this is fine.

File: jekyll/_posts/2017-05-25-faster-command-line-tools-in-nim.md [hunk 4]
```diff
@@ -258,9 +258,8 @@ I cleared out all build artefacts (such as the `*.o` files for D and the `nimcac
...
 ## Conclusion
 
 And that was the end of my little test. It turns out that Nim can certainly hold its own when it comes to parsing CSV files.
 
-I'd be interested to see how things run on other machines and operating systems to see how things are across platforms. I'm also sure there are far better ways I could benchmark these implementations, which is why I've [posted the project publicly on GitHub](https://github.com/euantorano/faster-command-line-tools-in-nim) - I'd welcome any enhancements to any of the versions or the benchmarking scripts, and would certainly welcome some more results to see how things stack up!
\ No newline at end of file
+I'd be interested to see how things run on other machines and operating systems to see how things are across platforms. I'm also sure there are far better ways I could benchmark these implementations, which is why I've [posted the project publicly on GitHub](https://github.com/euantorano/faster-command-line-tools-in-nim) - I'd welcome any enhancements to any of the versions or the benchmarking scripts, and would certainly welcome some more results to see how things stack up!
```
Comment (euantorano) [inline jekyll/_posts/2017-05-25-faster-command-line-tools-in-nim.md]: Not sure what the difference is here?

File: jekyll/_posts/2017-05-25-faster-command-line-tools-in-nim.md (hunk 1 above)
Comment (gangsterakato) [inline jekyll/_posts/2017-05-25-faster-command-line-tools-in-nim.md]: Yes, you're right.

File: jekyll/_posts/2017-05-25-faster-command-line-tools-in-nim.md (hunk 2 above)
Comment (gangsterakato) [inline jekyll/_posts/2017-05-25-faster-command-line-tools-in-nim.md]: I made this PR to fix that from the very start. So @dom96 you can close this PR and @euantorano I let you refactor your article with this fix.

File: jekyll/_posts/2017-05-25-faster-command-line-tools-in-nim.md (hunk 4 above)
Comment (gangsterakato) [inline jekyll/_posts/2017-05-25-faster-command-line-tools-in-nim.md]: None, just some Github weirdness.

File: jekyll/_posts/2017-05-25-faster-command-line-tools-in-nim.md (hunk 2 above)
Comment (euantorano) [inline jekyll/_posts/2017-05-25-faster-command-line-tools-in-nim.md]: @gangsterakato Ah, no problem. Sorry for my slowness, I was away all weekend and only got back last night.

**Pull request closed without merging**